scripts/get_version.py

```diff
@@ -0,0 +1,55 @@
+import requests
+from VSTSRequest import APIRequest
+
+def get_builds(response):
+    print(f'status code: {response.status_code}')
+    builds = []
+    for build in response.json()['value']:
+        builds.append(build['id'])
```

Comment:
return [build['id] for build in response.json()['value']]
List comprehensions are the best!

Reply:
I should have thought of this... 👍